### DISCUSSION TITLE
[Feature]: White Background for Pimcore SVG Thumbnails

### DISCUSSION THREAD

**Discussion by niederberger-qw** [pimcore/pimcore#13063] [Q&A / Support] [answered]
### Feature description

We have the problem, that our customer can not read his texts printed on the svgs.

![grafik](https://user-images.githubusercontent.com/85213755/188405757-cde54939-c2d5-4f4c-8b38-fffb7c9b4aee.png)

Maybe there is an option i don't know or we need an option to set a backgroundcolor for the transparent background within the pimcore backend thumbnail generation.

**Comment by dvesh3** [comment]
@niederberger-qw Hi, I moved the issue to discussions as this is the right place to get help from the community.

If there are clipping paths on your images then it is cropped automatically while generating thumbnail. Please try disabling it https://pimcore.com/docs/pimcore/current/Development_Documentation/Assets/Working_with_Thumbnails/Image_Thumbnails.html#page_Clipping-Support

**Comment by brusch** [comment, marked as answer]
You can use custom thumbnail definitions for the admin previews: 
https://github.com/pimcore/pimcore/pull/7437#issuecomment-720560199

**Comment by niederberger-qw** [comment]
Thank you very very much :)
